packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryConfirmKey/index.tsx

```diff
@@ -191,116 +208,111 @@ const AccountRecoveryConfirmKey = (_: RouteComponentProps) => {
       checkRecoveryKey(submitData);
     }
   };
-
-  if (linkStatus === LinkStatus.damaged || requiredParams === null) {
-    return <ResetPasswordLinkDamaged />;
-  }
-
-  if (linkStatus === LinkStatus.expired) {
+  if (showLoadingSpinner) {
     return (
-      <LinkExpiredResetPassword
-        email={requiredParams.email}
-        {...{ viewName }}
-      />
+      <LoadingSpinner className="bg-grey-20 flex items-center flex-col justify-center h-screen select-none" />
     );
-  }
+  } else {
```

Comment:
You don't need an `else` here, you can just `return`, since if it's truthy something else is returned. (That might also help with the diff in this section)